src/libstd/fs.rs

```diff
@@ -844,6 +844,9 @@ pub fn symlink_metadata<P: AsRef<Path>>(path: P) -> io::Result<Metadata> {
 
 /// Rename a file or directory to a new name.
 ///
+/// Moving a file or directory is also a rename, in a sense, and so this
+/// function is not limited a single directory.
```

Comment:
Hm, was the issue misread perhaps? This seems somewhat confusing as the main docs say "Rename" and then this says it's also a rename. Perhaps this could mention that it could move many files (as part of renaming a directory), but I think the issue in question was also asking for clarification that moving across filesystem boundaries probably isn't allowed.